projects/RabbitMQ.Client/client/impl/MethodArgumentReader.cs

```diff
@@ -45,32 +45,34 @@
 
 namespace RabbitMQ.Client.Impl
 {
-    internal struct MethodArgumentReader
+    internal ref struct MethodArgumentReader
```

Comment:
Should we ditch `ReadContent` since it says

```
throw new NotSupportedException("ReadContent should not be called");
```

Reply:
I'm fine with ditching it, I left it as it was there before.